examples/waves/CMakeLists.txt

```diff
@@ -0,0 +1,71 @@
+cmake_minimum_required(VERSION 3.10.2 FATAL_ERROR)
+project(ignition-rendering-waves)
+
+include_directories(SYSTEM
+  ${PROJECT_BINARY_DIR}
+)
+
+find_package(ignition-rendering6)
+
+set(TARGET_THIRD_PARTY_DEPENDS "")
+
+if (APPLE OR UNIX)
+  find_package(GLUT REQUIRED)
+  include_directories(SYSTEM ${GLUT_INCLUDE_DIRS})
+  link_directories(${GLUT_LIBRARY_DIRS})
+
+  find_package(OpenGL REQUIRED)
+  include_directories(SYSTEM ${OpenGL_INCLUDE_DIRS})
+  link_directories(${OpenGL_LIBRARY_DIRS})
+  set(TARGET_THIRD_PARTY_DEPENDS
+    ${TARGET_THIRD_PARTY_DEPENDS}
+    ${OPENGL_LIBRARIES}
+    ${GLUT_LIBRARIES}
+  )
+  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wno-deprecated-declarations")
+endif()
+
+if (NOT APPLE)
+  find_package(GLEW REQUIRED)
+  if (WIN32)
+    set(TARGET_THIRD_PARTY_DEPENDS
+      ${TARGET_THIRD_PARTY_DEPENDS}
+      GLEW::glew
+    )
+  else ()
+    set(TARGET_THIRD_PARTY_DEPENDS
+      ${TARGET_THIRD_PARTY_DEPENDS}
+      GLEW
+    )
+  endif()
+endif()
+
+configure_file (example_config.hh.in ${PROJECT_BINARY_DIR}/example_config.hh)
```

Comment:
I'm getting a compilation error from missing the `example_config.hh.in` in the waves example directory and I fixed it temporarily by just taking the file from the other examples.

Reply:
oops, added file in 56309b1